openfl-tutorials/experimental/workflow/1001_Workspace_Creation_from_JupyterNotebook.ipynb

```diff
@@ -946,7 +946,7 @@
     "The following cells convert the Jupyter notebook into a Python script and create a Template Workspace that can be utilized by Aggregator based Workflow\n",
     "> NOTE: Only Notebook cells that were marked with `#| export` directive shall be included in this Python script\n",
     "\n",
-    "We first import `WorkspaceExport` module and execute `WorkspaceExport.export()` that converts the notebook and generates the template workspace. User is required to specify: \n",
+    "We first import `WorkspaceExport` module and execute `WorkspaceExport.export_agg_based_workspace()` that converts the notebook and generates the template workspace. User is required to specify: \n",
```

Comment:
As a user facing function, the name change to `export_agg_based_workspace` is quite long and requires some digging to figure out what an Aggregator Based Workflow is. I would recommend this is changed back to `export` for now.  

In the future it would be nice to refactor these capabilities into a `NotebookTools` namespace (or `NotebookUtils`) so that it's very clear from looking at the class + function name what the purpose is.

`NotebookTools.export_workspace(...)`

Reply:
Reverted `export_agg_based_workspace` to `export` as per your recommendation.
Thank you for the suggestion! Refactoring these capabilities into a `NotebookTools` (or `NotebookUtils`) namespace makes a lot of sense for clarity and maintainability. We’ll add this to our backlog and aim to address it in a subsequent PR/release.